Documentation/dnx-instructions.md

```diff
@@ -0,0 +1,179 @@
+
+These instructions are specific to Windows and assume that you've just completed the [Windows installation instructions](https://github.com/dotnet/coreclr/wiki/Windows-instructions). They use the same demo `C:\coreclr-demo` directory and assume the same artifacts. The same general idea will work on OS X and Linux, once .NET Core is supported more broadly on those OSes.
+
+Introduction to DNX
+===================
+
+DNX is a .NET Execution Environment. It's a concept that came out of the ASP.NET 5 project, but is a more general concept than Web. DNX can be looked at a few different ways:
+
+- It is a distribution of .NET Core.
+- It delivers a source-first and NuGet-first .NET programming experience.
+- It is console-everything.
+- It is the base mechanism and set of concepts on which ASP.NET 5 is built.
+
+You can have more than one DNX installed (under `.dnx`) on a machine will have a single one selected for use at any time. You use [DNVM](https://github.com/aspnet/dnvm), the DNX Version Manager, to acquire and manage DNX instances.
+
+DNX instances differ/disambiguate on the following axis:
+
+- Version
+- CPU architecture
+- Runtime type (CLR, CoreCLR, Mono)
+
+Installing DNVM
+===============
+
+You need DNVM, the version manager, as a starting point. DNVM enables you to acquire a (or multiple) DNX. The easiest installation approach is to follow the [DNVM instructions](https://github.com/aspnet/home#install-the-net-version-manager-dnvm) on the ASP.NET home repo. The instructions provide the following powershell script to install DNVM on Windows.
+
+	C:\coreclr-demo>@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"
+
+DNVM is simply a script. It doesn't depend on or include a DNX or CoreCLR.
+
+You can request to see which DNXs are available, with `dnvm list`, which will display an empty set of installed runtimes.
+
+	C:\coreclr-demo>dnvm list
+
+Installing a DNX
+================
+
+You install a DNX via DNVM. It's easy to install the lastest default DNX, using the upgrade command.
+
+	C:\coreclr-demo>dnvm upgrade
+
+Alternatively, you can use the install command:
+
+	C:\coreclr-demo>dnvm install latest
+
+Both commands will install a default DNX, which on Windows, is x86 and hosted on the .NET Framework CLR. You can see that with `dnvm list`.
+
+	C:\coreclr-demo>dnvm list
+
+	Active Version           Runtime Architecture Location                       Al
+	                                                                             ia
+	                                                                             s
```

Comment:
This what "Alias" looks like on my screen. So, currently, the markdown is a faithful representation of what dnvm produces in a standard windows prompt. It is a bit odd, but that's what it looks like. Sound good to leave it as-is?